WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -134,12 +221,28 @@ class SiteDesignContentCollectionViewController: CollapsableHeaderViewController
     }
 
     private enum TextContent {
-        static let mainTitle = NSLocalizedString("Choose a theme", comment: "Title for the screen to pick a theme and homepage for a site.")
-        static let backButtonTitle = NSLocalizedString("Design", comment: "Shortened version of the main title to be used in back navigation.")
-        static let skipButtonTitle = NSLocalizedString("Skip", comment: "Continue without making a selection.")
-        static let cancelButtonTitle = NSLocalizedString("Cancel", comment: "Cancel site creation.")
-        static let errorTitle = NSLocalizedString("Unable to load this content right now.", comment: "Informing the user that a network request failed becuase the device wasn't able to establish a network connection.")
-        static let errorSubtitle = NSLocalizedString("Check your network connection and try again.", comment: "Default subtitle for no-results when there is no connection.")
+        static let mainTitle = NSLocalizedString("Choose a theme",
+                                                 comment: "Title for the screen to pick a theme and homepage for a site.")
+        static let backButtonTitle = NSLocalizedString("Design",
+                                                       comment: "Shortened version of the main title to be used in back navigation.")
+        static let skipButtonTitle = NSLocalizedString("Skip",
+                                                       comment: "Continue without making a selection.")
+        static let cancelButtonTitle = NSLocalizedString("Cancel",
+                                                         comment: "Cancel site creation.")
+        static let errorTitle = NSLocalizedString("Unable to load this content right now.",
+                                                  comment: "Informing the user that a network request failed becuase the device wasn't able to establish a network connection.")
```

Comment:
I noticed there's small pre-existing typo (becuase), but I guess this probably doesn't matter too much, since it's only the translation context :man_shrugging: .